neural_compressor/torch/quantization/autotune.py

```diff
@@ -93,17 +93,19 @@ def autotune(
         tuning_logger.trial_end(trial_index)
         if tuning_monitor.need_stop():
             logger.info("Stopped tuning.")
-            del q_model  # maybe gc.collect() is needed for memory release
-            best_quant_config: BaseConfig = tuning_monitor.get_best_quant_config()
-            # !!! Make sure to use deepcopy only when inplace is set to `True`.
-            q_model = quantize(
-                deepcopy(model),
-                quant_config=best_quant_config,
-                run_fn=run_fn,
-                run_args=run_args,
-                inplace=True,
-                example_inputs=example_inputs,
-            )
+            if trial_index == 0:  # recover the best q_model from previous results.
```

Comment:
Good point, However, we might need to refine the condition a bit further. It would be more robust to check whether the `best_trial_index` is equal to the `last_trial_index`. If so, we can skip the recovery process. `trial_index == 0` is one such case.

Reply:
yeah, we can target it at next release